common/src/main/java/com/graphicsfuzz/common/util/IRandom.java

```diff
@@ -24,12 +24,19 @@
 
   int nextInt(int bound);
 
+  default int nextInt(int origin, int bound) {
```

Comment:
Please add a JavaDoc comment indicating the range of integer that could be returned - i.e., making it clear that bound should be larger than origin, and that an integer in the range [origin, origin + bound) is returned.